### ISSUE TITLE
Mod Loader not recognized 

### ISSUE THREAD

**Issue by paulkie** [xforce/anno1800-mod-loader#184]
With the release of 0.9.4, whenever I'm launching Anno 1800, the message 'modding detected' does not appear and I'm unable to play with mods. I've tried verifying files and installing 0.9.4 again and it didn't work. Uninstalling and reinstalling the game doesn't work either. I also tried playing only with mods that were updated for GU16, didn't work. Any idea on what I can do? Hope I'm not reporting a known bug.

**Comment by TocsNo666** [comment]
I found a work around by using a mod loader. Found it on readit and its called "I Mod Your Anno" (I think i just typed Anno mod manager in readit). I deleted all the modded files and took it back to the base game then i downloaded to Mod loader through this program it will update the puthon35 files and create a folder for the mods for you. This program also has 6 collections of mods that can be downloaded to start the modded experience but you can add more the usual way. 

hopes it helps as it did me after pulling my hair out for last few weeks

**Comment by jasonbuff** [comment]
Yes verify game files ubisoft by right clicking game download mod loader update it then download the update python files and install in the correct spot as indicated in the instructions

**Comment by paulkie** [comment]
@jasonbuff what do you mean update the mod loader? How can you update two .dll files? I've already verified the files three times and mods aren't recognized

**Comment by Mannie313** [comment]
bei mir geht die 0.9.4 sehr gut. Muß nur noch ein paar mods aussortieren und dann kann das Spielen losgehen.

**Comment by Gunslinger32** [comment]
It,s seems with a new game it works fine, but not with saved games

**Comment by paulkie** [comment]
@Gunslinger32 I just tried that and it didn't work. It still doesn't say 'modding detected' in the title screen and no mod seems to be working when I play.

**Comment by cos155** [comment]
> @Gunslinger32 I just tried that and it didn't work. It still doesn't say 'modding detected' in the title screen and no mod seems to be working when I play.

Hi @Gunslinger32  - I got mine working by verifying my files, then by re-downloading and replacing the python files.  Have to add - it has just crashed on me when I tried to build a fracking station...... hopefully it isn't that that has caused it :(

**Comment by OneklickLP** [comment]
I have the same issue. Reverifing and reinserting, restarting, reinstalling C++

Nothing that i did helped.

I copied the two .dlls into the anno.exe folder (win64).

I have the Steam version, with almost all DLCs.

pls help. I never modded before, so this is isn't a modding problem. this is a more broad installation problem imho.

**Comment by OneklickLP** [comment]
> With the release of 0.9.4, whenever I'm launching Anno 1800, the message 'modding detected' does not appear and I'm unable to play with mods. I've tried verifying files and installing 0.9.4 again and it didn't work. Uninstalling and reinstalling the game doesn't work either. I also tried playing only with mods that were updated for GU16, didn't work. Any idea on what I can do? Hope I'm not reporting a known bug.

Hey mate. Can we maybe check if we have a something in common to pin down the issue? (comparing hardware and software)

For example im running
Windows 11 22H2 on a 6900XT, B550 Motherboard, R9 5900X and Anno 1800 is installed on Steam.

I've already figured out that it doesnt affect Steam in general but maybe if we exchange build details we can figure out why we booth have the same issue!

A reply is much appreciated
